bin/table_classification_summaries_main

```diff
@@ -111,7 +111,7 @@ def main(data_dcts, window_size, latex_file, with_preamble=False):
                     median_t, error_t = bootstrap_confidence_interval(time_to_steadys)
                     time_to_steady = format_median_error(median_t, error_t, brief=True)
                 else:  # No changepoints in any process executions.
-                    mean_last_cpt = format_median_error(median, 0, as_integer=True)
+                    mean_last_cpt = format_median_error(0, 0, as_integer=True)
```

Comment:
It isn't a subtle bug, but unfortunately GitHub has folded the relevant line of code. Before the if/else, `median` is set to the median of the last segment means, on line 108, so this is a mean time, not a mean iteration, but you wouldn't notice because it is printed as an integer.